### PR TITLE
Improved UI for Todo Items

### PR DESCRIPTION
**Pull request by soustab10**
### Changes made:
- [x] Improved UI of the Todo Items across all screen sizes.
- [x] Custom width of Todo items across screen sizes for better rendition.

Mobile:
image

Desktop:
image

If approved, @Rick-mad-lab Please merge it. Thank you.

### CODE REVIEW

**Comment by Rick-mad-lab** [comment]
🙂 nice 👍

**Comment by soustab10** [comment]
Please add hacktoberfest-accepted label. Thank you.

File: style.css [hunk 1]
```diff
@@ -3,10 +3,10 @@ body {
   font-size: 16px;
   color: #e5e0e0;
   background-image: linear-gradient(to bottom, #655d8a 30%, #7897ab 70%);
-  font-family: 'Poppins', sans-serif;
+  font-family: "Poppins", sans-serif;
```
Comment (Rick-mad-lab) [inline style.css:6]: ?

**Review by Rick-mad-lab — COMMENTED** [review]
done

**Comment by Rick-mad-lab** [comment]
@soustab10, after this update this happen sorry I have to revert this PR.
Screenshot 2022-10-04 at 8 21 41 AM

**Pull request merged**